server/commands.lua

```diff
@@ -1,15 +1,70 @@
------------------------
-----    Imports    ----
------------------------
-
 local functions = require 'server.functions'
-
-local removeKeys = functions.removeKeys
 local giveKeys = functions.giveKeys
 
------------------------
-----   Commands    ----
------------------------
+---@param src number
+---@param vehicle number
+---@return boolean
+local function hasKeys(src, vehicle)
+    local keysList = Player(src).state.keysList or {}
+    local sessionId = Entity(vehicle).state.sessionId
+    return keysList[sessionId]
+end
+
+exports('HasKeys', hasKeys)
+
+---@param src number
+---@return number?
+local function getClosestPlayer(src)
```

Comment:
Why do we still have this instead of `lib.getClosestPlayer`?

Reply:
lib.getClosestPlayer on the server will always return the player. What we want is basically the second closest player from the players coordinates.

Reply:
Ah, I see.